src/main/java/com/baro/auth/infra/jwt/JwtTokenDecrypter.java

```diff
@@ -0,0 +1,43 @@
+package com.baro.auth.infra.jwt;
+
+import com.baro.auth.exception.jwt.JwtTokenException;
+import com.baro.auth.exception.jwt.JwtTokenExceptionType;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.SecretKey;
+
+@RequiredArgsConstructor
+@Component
+class JwtTokenDecrypter implements TokenDecrypter {
+
+    private final JwtProperty jwtProperty;
+
+    public Long decrypt(String authHeader) {
```

Comment:
authHeader라는 모호한 view용어보단 `decrypt()`역할에 맞는 인증 정보 `authroization`은 어떨까요
```suggestion
    public Long decrypt(String authorization) {
```